documentation/meta/incidents/index.md

```diff
@@ -0,0 +1,154 @@
+  researched. Stabilized but not yet resolved
+- **Resolved** - The incident is stable, and long-term fixes to prevent future
+  occurrences or other mitigations have been identified.
+
+As you might expect, incidents can be _destabilized_ if they reoccur, and the
+state should rollback to "Stabilization pending":
+
+```{mermaid}
+graph LR
+    A[Stabilization pending]
+    B[Stabilized]
+    C[Under investigation]
+    D[Resolved]
+
+    A --> B
+    B --> C
+    B --> A
+    C --> A
+    C --> D
+    D --> A
+```
+
+## How do we _handle_ incidents?
+
+```{warning}
+Dealing with an incident now? Go to the [Incident Response Runbook](/meta/incidents/runbooks/incident_response_runbook).
+```
+
+This section addresses the philosophy of our approach to incident management and
+explanation behind the steps in the
+[runbook](/meta/incidents/runbooks/incident_response_runbook).
+
+There are four main steps to handling production incidents:
+
+<!-- prettier-ignore -->
+| # | Name | Goal | Status After |
+|---|------|------|--------------|
+| 1 | Record | Visibly acknowledge the incident and create the communication threads necessary for ongoing discussion and investigation. Understand how bad the situation is based on our severity definitions. | Stabilization pending OR Stabilized (as appropriate)
+| 2 | Stabilize |<p>If there is a total or partial outage, identify service stabilization options and action them. | Stabilized |
+| 3 | Resolve | Identify outstanding improvements to the service that would mitigate the issue in the future and create tickets to implement. | Under investigation |
+| 4 | Review | Identify process improvements. If the underlying cause is still unknown, identify further steps to resolve or continue investigation of the issue via 5-whys. | Under investigation OR Resolved (as appropriate) |
+
+All maintainers are required to report incidents they encounter. **Crucially,
+maintainers who record incidents are not _inherently_ responsible for
+stabilizing them.** Their only required job is to record the incident. Every
+incident has a **Lead** who is the person responsible for providing updates on
+the incident and delegating tasks. The Lead is also responsible for hosting the
+retrospective after the incident is resolved.
+
+### How do we determine the lead?
+
+By default, this is the week’s MSR runner. If it is outside of the MSR runner’s
+working hours or they are unresponsive (with later explanation, ideally) someone
+else will need to be the lead. It is the recorder’s job to find this lead or act
+as this lead. Factors which should be considered in finding a lead are
+availability and expertise. Is anyone online uniquely qualified to take this
+work on? Alternatively, has one person been handling most incidents and this
+time, someone new should take on? Consulting the threads for past incidents will
+make it easier to see the distribution of incident leadership so we can easily
+find who hasn’t done it recently.
```

Comment:
Over time we'll come to understand this better, but a useful clarification here could be that stabilisation of >= severity 2 incidents _probably_ takes priority over finding a lead, if the de facto lead is unavailable. Mostly, I want to avoid the team or any individual on the team to accidentally think they both need to try to work on stabilisation _and_ find a lead (e.g., that they are simultaneous priorities and expectations) _or_ risk the assumption that the person leading stabilisation for critical issues is also necessarily the lead.